examples/unixfs-file-cid/main_test.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"github.com/ipfs/go-cid"
+	"testing"
+
+	"github.com/libp2p/go-libp2p/core/peer"
+)
+
+func TestBitswapFetch(t *testing.T) {
```

Comment:
@guseggert any thoughts on how you'd like the testing to work here? Would you prefer something that runs the binary itself, reads logs as in https://github.com/libp2p/go-libp2p/blob/34dc11547a10e7abe0a3fc615dbf839818554cab/examples/echo/main_test.go#L11, or something else?